torch/csrc/jit/ir.h

```diff
@@ -1267,20 +1263,11 @@ struct PythonOp : public Node {
   // used in ONNX for discovering symbolics
   virtual c10::optional<THPObjectPtr> autogradFunction() const = 0;
 
-  // should this Python function be skipped over when exported (i.e. for
-  // debugging functions that only run in Python)
-  bool ignore_on_export = false;
+  virtual void lint_python() const = 0;
 };
 // patched in when python bindings are loaded
 TORCH_API PythonOp* allocPythonOp(Graph* g);
 TORCH_API void setAllocPythonOp(PythonOp* (*v)(Graph* g));
-inline Node* Graph::createPythonOp(
-    THPObjectPtr&& pyobj,
-    const std::string& cconv,
-    pyobj_list&& scalar_args) {
-  PythonOp* op = allocPythonOp(this);
```

Comment:
allocPythonOp and setAllocPythonOp are now dead. Can we delete them?

Reply:
Yeah, I wasn't sure whether we might use them for something else, always happy to :)